### ISSUE TITLE
Invisible series is still focusable

### ISSUE THREAD

**Issue by pvsleeper** [highcharts/highcharts#16634]
#### Expected behaviour
When a series is not visible, it should not be focusable, otherwise it gives the user the experience that focus is lost.

#### Actual behaviour
When a11y is enabled and the you click on the legend to remove the series, the removed series is still focusable.

#### Live demo with steps to reproduce
https://jsfiddle.net/wxch4rbj/4/

#### Product version
9.3.1

#### Affected browser(s)
Google Chrome, Edge

**Comment by pvsleeper** [comment]
Hmmm... it seems like this may have been done on purpose. Guessing this is so that the screen reader has something to focus on to read out the chart title etc, even if there is no data.

**Comment by KacperMadej** [comment]
Hi @pvsleeper 

Thank you for reporting the issue.

**Comment by oysteinmoseng** [comment]
Can confirm that this is a bug and not intended behavior, we only want interactive elements to receive kbd focus.

**Comment by oysteinmoseng** [comment]
Fixed in pending PR #16732. The fix should be part of the next official release.

Closing this issue manually as our automation won't catch it when multiple issues are fixed in one PR.
